cws-service/src/main/java/jpl/cws/process/initiation/aws/S3DataManager.java

```diff
@@ -26,25 +26,30 @@
  * @author ghollins, jwood, ztaylor
  */
 public class S3DataManager {
```

Comment:
```suggestion
public class S3DataManager implements AutoClosable {
```
I'd suggest we instead make this class AutoClosable, so that it'll get cleaned up even in sneaky places we don't expect. It also allows us to use the [try with resources](https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html) paradigm instead of `try...finally`.